module/VuFind/src/VuFind/RecordDriver/SolrOverdrive.php

```diff
@@ -581,6 +591,7 @@ public function getURLs()
     {
         return $this->getIsMarc()
             ? parent::getURLs()
+            // ? $this->getPermanentLink()
```

Comment:
Is this a stray comment that wasn't meant to be committed?

Reply:
Yup, fixed.